app/src/examples/RuntimeTests/ReanimatedRuntimeTestsRunner/Matchers.ts

```diff
@@ -45,6 +45,26 @@ export class Matchers {
     };
   };
 
+  private _toBeWithinRangeMatcher: MatcherFunction = (
+    currentValue: TestValue,
+    minimumValue: unknown,
+    maximumValue: unknown,
+  ) => {
+    const currentValueAsNumber = Number(Number(currentValue));
+    const validInputTypes = typeof minimumValue === 'number' && typeof maximumValue === 'number';
+    const isWithinRange = Number(minimumValue) <= currentValueAsNumber && currentValueAsNumber <= Number(maximumValue);
+
+    const coloredExpected = color(`[${minimumValue}, ${maximumValue}]`, 'green');
+    const coloredReceived = color(currentValue, 'red');
+
+    return {
+      pass: isWithinRange && validInputTypes,
```

Comment:
So if someone creates a test with wrong input types, they will see it as not passing test. Hmmm.. wouldn't it be better to just force all `currentValue`, `minimumValue` and `maximumValue` arguments to be `number` types? Then it forces test suite users to use correct types because the API says so.

Reply:
Sure! I've fixed all the types